### PR TITLE
Py brench

### PR DESCRIPTION
**Pull request by wuyntmm**
Homework #4

### CODE REVIEW

File: hw4.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+text = input('Enter the line ')
+if len(text) != 0:
+    if text.isdigit():
+        print('You have entered a number and ', end='')
+        if int(text) % 2 == 0:
+            print('your number is even')
+        else:
+            print('your number is odd')
+    else:
+        length = len(text)
```
Comment (klymenok) [inline hw4.py:10]: You can put `len(text)` directly into curly brackets in the f-string below

File: hw3.py [hunk 2]
```diff
@@ -1,22 +1,23 @@
-print('I love Python '*42)
+print('I love Python ' * 42)
 
 age_in_month = 327
 age_in_years = age_in_month // 12
 print(age_in_years)
 
-name='Nazar'
-my_age="My name is " + name + " I'm " + str(age_in_years) + " years old"
+name = git
```
Comment (klymenok) [inline hw3.py]: Changes in this file are not required by the task. I can approve it, bu only after fixing some error in this line: `name = git`

**Review by klymenok — CHANGES_REQUESTED** [review]
Homework is done well, but there is some changes of previous homework that I cannot accept with mistakes

**Comment by wuyntmm** [comment]
Homework3 fixed!

**Review by klymenok — APPROVED** [review]
Good job!

**Pull request merged**